### PR TITLE
Swap array element if Prime

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/swappingArrayElementREADME.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Input : [1, 2, 3, 7]
+Output: 7 is a prime number.
+Original Array is: [1, 2, 3, 7]
+Swapped array is: [7, 2, 3, 1]
+
+Input : [1, 2, 3]
+Output: 3 is a prime number.
+Original Array is: [1, 2, 3]
+Swapped array is: [3, 2, 1]
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/swappingArrayElementREADME.md]: This inputs are not given in code

File: src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/swappingArrayElementREADME.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+Input : [1, 6, 7, 9, 8]
+Output: 
+
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/swappingArrayElementREADME.md]: Not used

File: src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/SwappingArrayElement.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+public class SwappingArrayElement {
+    public static boolean isPrime(int number) {
+        int index;
+        for (index = 2; index < number / 2; index++) {
+            if (number % index == 0) {
+                return false;
+            }
+        }
+        return true;
+    }
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/SwappingArrayElement.java]: Please correct the loop condition

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/SwappingArrayElement.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package MediumComplexityProblems.swappingArrayElement;
+
+import java.util.Arrays;
+
+public class SwappingArrayElement {
+    public static boolean isPrime(int number) {
+        int index;
+        for (index = 2; index <= number / 2; index++) {
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/MediumComplexityProblems/swappingArrayElement/SwappingArrayElement.java:8]: return true in if condition when number is prime

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Pull request merged**